packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -815,11 +822,24 @@ function commitUnmount(
           current.stateNode = null;
         }
       }
+      return;
+    }
+    case DehydratedFragment: {
+      if (enableSuspenseCallback) {
+        const hydrationCallbacks = finishedRoot.hydrationCallbacks;
+        if (hydrationCallbacks !== null) {
+          const onDeleted = hydrationCallbacks.onDeleted;
+          if (onDeleted) {
```

Comment:
`!== null`?

Reply:
This is an optional property. In some other cases we check `typeof ... === 'function'` when it's a prop, but in some cases we check truthiness. Not sure which one to be consistent with so I went with the one that takes fewer bytes.